src/mono/wasm/debugger/tests/debugger-test/debugger-evaluate-test.cs

```diff
@@ -508,58 +508,72 @@ public class EvaluateLocalsWithIndexingTests
     {
         public record Indexer(int index);
 
-        public class TestEvaluate
+        public class CommonCollections
         {
-            public List<int> numList;
-            public List<string> textList;
-            public int[] numArray;
-            public string[] textArray;
+            public List<int> numList = new List<int> { 1, 2 };
+            public List<string> textList = new List<string> { "1", "2" };
+            public int[] numArray = new int[] { 1, 2, 0 };
+            public string[] textArray = new string[] { "1", "2" };
             public int[][] numArrayOfArrays;
             public List<List<int>> numListOfLists;
             public string[][] textArrayOfArrays;
             public List<List<string>> textListOfLists;
-            public Dictionary<string, bool> indexedByStr;
-            public Dictionary<char, string> indexedByChar;
-            public Dictionary<bool, string> indexedByBool;
-            public int idx0;
-            public int idx1;
-
-            public string this[char key] => "res_" + key;
-            public string this[bool key] => key.ToString();
-            public bool this[string key] => key.Length > 3;
-            public int this[double key] => (int)key;
-            public int this[float key] => (int)key;
-            public int this[decimal key] => (int)key;
-            public int this[Indexer indexer] => indexer.index;
-            public char this[char[] arr] => arr.Length == 0 ? '0' : arr[0];
-
-            public double this[int key1, double key2] => key1 + key2;
-            public string this[char key1, string key2, string key3] => $"{key1}-{key2}-{key3}";
+            public Dictionary<string, bool> indexedByStr = new Dictionary<string, bool>() { { "1", true }, { "111", false }, { "true", true} };
```

Comment:
note: be careful with changing existing test code, as it can be easy to break tests even when we think we changed the line numbers correctly, especially if they end up affecting line numbers used for breakpoints for instance. Such a change might not break a test immediately, but it might now be stopping on the incorrect line opening a potential for the test not doing the intended checks.